ui/components/app/app-header/index.scss

```diff
@@ -81,7 +81,6 @@
 
   &__account-menu-container {
     display: flex;
-    flex-flow: row nowrap;
```

Comment:
is this because `row nowrap` is default?

Reply:
![Screen Shot 2022-03-09 at 2 22 54 PM](https://user-images.githubusercontent.com/34557516/157529420-388cb0ea-48a8-4986-bd5d-0f21bcb524ee.png)
yes that seems to be it

Reply:
There were many cases I found in which selectors listed duplicate properties (not sure why this wasn't picked up by stylelint before). This is one of them (note how `flex-flow` is already listed on line 87).

Reply:
Oh ok yeah I over thought it in this case. But yeah I noticed a lot of the duplication getting cleaned-up otherwise.